runtime/block_quantization_kernels.cu

```diff
@@ -130,6 +130,8 @@ __device__ void block_quantize_to_nvfp4(
 
   if constexpr (USE_GLOBAL_SCALE) {
     scaled_max = global_scale[0] / scaled_max;
+  } else {
+    scaled_max = 1.0 / scaled_max;
```

Comment:
This should be the right math, but looks like it's causing cpp test errors. I'll double check that.